cime_config/config_grids.xml

```diff
@@ -4540,6 +4638,11 @@
       <map name="ROF2OCN_LIQ_RMAPNAME">cpl/cpl6/map_rx1_to_ECwISC30to60E3r2_smoothed.r150e300.230901.nc</map>
     </gridmap>
 
+    <gridmap ocn_grid="IcoswISC30E3r2" rof_grid="rx1">
+      <map name="ROF2OCN_ICE_RMAPNAME">cpl/cpl6/map_rx1_to_IcoswISC30E3r2_smoothed.r150e300.230905.nc</map>
```

Comment:
"smoothed" should be "cstmnn"  for "custom nearest neighbor".

Reply:
should be fixed now -- I also changed the datestamp to YYYYMMDD to match the new naming convention